Resources/doc/reference/action_list.rst

```diff
@@ -148,6 +148,12 @@ Available types and associated options
 +-----------+----------------+-----------------------------------------------------------------------+
 | datetime  | format         | A format understandable by Twig's ``date`` function.                  |
 +-----------+----------------+-----------------------------------------------------------------------+
+| email     | as_string      | Renders as email link                                                 |
```

Comment:
The description is wrong too me.

If the `as_string` parameter is set to `true`, it deactivate the email link.

I would better write: 

```rst
Disables the email link if set to ``true``.
```